site/_includes/primary-nav-items.html

```diff
@@ -10,6 +10,6 @@
     <li class="training-menu"><a href="/news/">News</a>
     </li>
     <li>
-        <button class="button-default button-green trigger-top-form"> <a href="https://github.com/apache/incubator-metron/releases" target="new">Download </a></button>
+        <button class="button-default button-green trigger-top-form"> <a href="https://dist.apache.org/repos/dist/release/incubator/metron/" target="new">Download </a></button>
```

Comment:
Hi Kyle, this change is on the right path, since we definitely should be publishing downloads on "dist".  However, http://www.apache.org/dev/release-download-pages.html says:
- All links to the distribution artifacts **must not** reference the main Apache web site ["dist"]. They should use the standard mechanisms to distribute the load between the mirror sites [see http://www.apache.org/dyn/closer.cgi/ ].
- All links to checksums, detached signatures and public keys **must** reference the main Apache web site and should use https (SSL).

The web page then goes on in some detail about how to set up a script to automatically redirect the user to an appropriate mirror.  

If the above isn't clear, there is a better written explanation at http://incubator.apache.org/guides/releasemanagement.html#understanding-mirroring
But it points back to http://www.apache.org/dev/release-download-pages.html to talk about the script.

An example of a good download page that uses an automatic mirror-direction script is
https://maven.apache.org/download.cgi
An example of a download page that is "dumber" but therefore easier to set up is
https://hadoop.apache.org/releases.html

The quick-and-dirty fix is to simply point at http://www.apache.org/dyn/closer.cgi/incubator/metron/ instead of 
https://dist.apache.org/repos/dist/release/incubator/metron/ , BUT that doesn't give the user access to the secure signatures, and that's not acceptable.